### PR TITLE
Add a note about iOS behavior

### PR DESCRIPTION
**Pull request by stoyan**
To test the behavior with the workaround: https://www.onlinemusictools.com/drone/
Or without the workaround: https://www.onlinemusictools.com/drone/safari.html
For example: 
1. start the click/metronome
2. stop it
3. minimize Safari
4. go back and start again

### CODE REVIEW

File: files/en-us/web/api/baseaudiocontext/state/index.html [hunk 1]
```diff
@@ -57,6 +57,21 @@ <h2 id="Specifications">Specifications</h2>
  </tbody>
 </table>
 
+<h2 id="iossafari">iOS Safari</h2>
```
[the remark this answers is not in the document]
Comment (chrisdavidmills) [reply files/en-us/web/api/baseaudiocontext/state/index.html:60]: And could you also update the "Example" heading text and ID to "Examples"? ;-)

**Review by chrisdavidmills — CHANGES_REQUESTED** [review]
Hi @stoyan ! Thanks for this useful addition. Just a couple of small requests for change, after which it'll be ready to merge.

File: files/en-us/web/api/baseaudiocontext/state/index.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (stoyan) [reply files/en-us/web/api/baseaudiocontext/state/index.html]: Thanks Chris! Addressed the comments and also found a typo

**Review by chrisdavidmills — APPROVED** [review]
Great, thanks @stoyan . I've just tweaked the title IDs so they match the titles (this is what MDN has always done, for better or for worse), and now we are ready to merge.

**Pull request merged**